index.js

```diff
@@ -1,3 +1,4 @@
+export { MINER_TO_PEERID_CONTRACT_ADDRESS, ABI } from './lib/constants.js'
```

Comment:
I find the name `ABI` too generic. Can we prefix it the same way the contract address is prefixed?

```suggestion
export { MINER_TO_PEERID_CONTRACT_ADDRESS, MINER_TO_PEERID_CONTRACT_ABI } from './lib/constants.js'
```

Reply:
Resolved in fcf0be2